gato/policy/gato_policy.py

```diff
@@ -48,9 +51,7 @@ def __init__(
 
         self.context_len = context_len
         self.pad_seq = pad_seq
-        # this is a dummy value as this implementation does not yet handle language IO
-        #self.text_tokens = 32000 # SentencePiece vocab size
-        self.text_tokens = 1
+        self.text_tokens = 50257 # gpt2
```

Comment:
make this configurable in the future e.g. decide this after you load GPT2 and then infer this